benches/ironcore_alloy_bench.rs

```diff
@@ -127,9 +115,82 @@ fn benches(c: &mut Criterion) {
     // }
 }
 
+fn tsp_benches(c: &mut Criterion) {
+    let rng = rand::thread_rng();
+    let tsp_uri = env::var("TSP_ADDRESS").unwrap_or("http://localhost".to_string());
+    let tsp_port = env::var("TSP_PORT").unwrap_or("32804".to_string());
+    let tenant_id = env::var("TENANT_ID").unwrap_or("tenant-gcp".to_string());
```

Comment:
I feel like we should just make this default to `tenant-gcp-l`. Adding the external calls just makes this so much less consistent